frontend/src/main/scala/game/Mouse.scala

```diff
@@ -27,7 +27,11 @@ final class Mouse(canvas: html.Canvas, controls: Controls) {
   private val mouseClickBus: EventBus[MouseEvent] = new EventBus
   val $mouseClicks: EventStream[MouseEvent]       = mouseClickBus.events
 
-  private val clickHandler: js.Function1[MouseEvent, _] = mouseClickBus.writer.onNext _
+  private val clickHandler: js.Function1[MouseEvent, _] = { (event: MouseEvent) =>
+    if (event.istrusted()) {
+      mouseClickBus.writer.onNext(event)
+    }
```

Comment:
Apparently `istrusted` is not a function and is spelled `isTrusted`.

So we can replace these lines with
```
if event.isTrusted then mouseClickBus.writer.onNext(event)
```